packages/idyll-layouts/src/blog/index.js

```diff
@@ -2,7 +2,10 @@ import createStyles from './styles';
 
 const config = {
   maxWidth: 600,
-  margin: '0 0 0 50px'
+  marginTop: 0,
+  marginRight: 0,
+  marginBottom: 0,
+  marginLeft: '50px'
 };
```

Comment:
The `centered` layout should also be updated to have the margins set more explicitly like this.

Reply:
Updated!